realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -267,6 +272,18 @@ public RxObservableFactory getRxFactory() {
         return rxObservableFactory;
     }
 
+    /**
+     * FIXME
+     * @return
+     */
+    @Nullable
+    public FlowFactory getFlowFactory() {
+        if (flowFactory == null) {
```

Comment:
We should guard against this in the builder so it can never be null here.

Reply:
Is this not the same case as with the Rx factory? It will be null if it isn't present in the classpath. By the way, I have just added a utility method for checking whether Flows are present or not.

Reply:
Oh, I see what you mean. Sure, I will mark the parameter as `@Nonnull` (for Kotlin's sake) and throw an exception in case we still receive null in Java.